src/test/java/Step3/RacingGameTest.java

```diff
@@ -0,0 +1,50 @@
+package Step3;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class RacingGameTest {
+
+    RacingGame racingGame = new RacingGame();
+
+    // params: carNum, playNum
+    @ParameterizedTest
+    @CsvSource({
+            "3, 5",
+            "5, 10",
+            "1, 40"
+    })
+    void car_list_length_is_equal_to_car_num(int carNum, int playNum) {
+        racingGame.makeBoard(carNum, playNum);
+
+        assertThat(racingGame.getCars().size()).isEqualTo(carNum);
```

Comment:
```suggestion
        assertThat(racingGame.getCars()).hasSize(carNum);
```
`hasSize` 메소드를 지원하고 있어 활용할 수 있겠네요 😄